src/client/parsiq-client.ts

```diff
@@ -59,8 +61,27 @@ class ParsiqClient {
     this.nftRequestHandler = new NftRequestHandler(apiKey, chain, config);
     this.balancesRequestHandler = new BalancesRequestHandler(apiKey, chain, config);
     this.web3HooksRequestHandler = new Web3HooksRequestHandler(apiKey, chain, config);
+    this.transactionLifecycleRequestHandler = new TransactionLifecycleRequestHandler(apiKey, chain, config);
   }
 
+  public readonly txLifecycle = {
+    create: (createTransactionLifecycle: CreateTransactionLifecycle) => {
+      return this.transactionLifecycleRequestHandler.createTransactionLifecycle(createTransactionLifecycle);
+    },
+
+    list: () => {
+      return this.transactionLifecycleRequestHandler.listTransactionLifecycles();
+    },
+
+    show: (id: string) => {
```

Comment:
I think in both cases, tx lifecycle and web3hooks the better naming is `get`?

Reply:
or let's just shorten Danny's options:

createTxLcRequest, listTxLcRequests...